lifecycle/lifecycle/deployer/redeploy.py

```diff
@@ -83,6 +83,51 @@ def reprovision_fatman(
         raise e
 
 
+def move_fatman(
+    fatman_name: str,
+    fatman_version: str,
+    new_infra_target: str,
+    config: Config,
+    plugin_engine: PluginEngine,
+    deployer_username: str,
+    auth_subject: Optional[models.AuthSubject],
+):
+    """Move fatman from one infrastructure target to another"""
+    fatman = read_fatman(fatman_name, fatman_version, config)
+    manifest = fatman.manifest
+    old_infra_target = fatman.infrastructure_target
+    assert fatman.manifest is not None, "fatman doesn't have Manifest data specified"
+    assert fatman.image_tag is not None, "fatman's image tag is unknown"
+    assert new_infra_target, "infrastructure target has to be specified"
```

Comment:
```suggestion
    assert new_infra_target is not None, "infrastructure target has to be specified"
```

I assume you want to assert some property (likely that it's not None) about `new_infra_target`?

Reply:
`assert new_infra_target` is stronger as it checks if it's neither None nor empty. See https://stackoverflow.com/a/9573283/6772197